pyproject.toml

```diff
@@ -63,6 +63,7 @@ extra-dependencies = [
   # LLMMetadataExtractor dependencies
   "amazon-bedrock-haystack>=1.1.1",
   "google-vertex-haystack>=2.0.0",
+  "ipython"
```

Comment:
I believe this was added for testing/debugging purposes? we should also remove it before the release

Reply:
No, it was added because the run tests require it (they test pipeline drawing which detects if the pipeline runs in a notebook or in a script.)

Reply:
can you add a simple comment on top so that we know why it's needed/who uses it